blockchain/vm/opcodes.go

```diff
@@ -103,6 +103,7 @@ const (
 	NUMBER
 	DIFFICULTY
 	GASLIMIT
+	CHAINID OpCode = 0x46
```

Comment:
This is same with below? @yoomee1313 
```suggestion
	CHAINID OpCode = 0x46
```

Reply:
@KimKyungup Yes, both expressions are the same.

Reply:
If so, why do you explicitly write `OpCode`? It is also different from geth.

Reply:
@ehnuje I referenced latest geth code. I should have mentioned it.. https://github.com/ethereum/go-ethereum/blob/d9c9ee5ac9631c434380f73b6e6e50030a04f8ee/core/vm/opcodes.go#L104